repositories.bzl

```diff
@@ -553,9 +555,8 @@ cc_proto_library(
         name = "googleapis_git",
         build_file_content = BUILD,
         patch_cmds = ["find . -type f -name '*BUILD*' | xargs rm"],
-        strip_prefix = "googleapis-c27d0463cc0097d7a64a7b108aca659df00d2e06",  # May 18, 2021
-        url = "https://github.com/googleapis/googleapis/archive/c27d0463cc0097d7a64a7b108aca659df00d2e06.tar.gz",
-        sha256 = "10333203467098e6af58242be3f780e509a09110a0d136ef7aba7fe75de33fd9",
+        strip_prefix = "googleapis-f49123a39efc18f0a20a2183629c62830725343c", # May 27, 2024
+        url = "https://github.com/googleapis/googleapis/archive/f49123a39efc18f0a20a2183629c62830725343c.tar.gz"
     )
 
```

Comment:
no sha265?

Reply:
This field is optional but after reading the document again (documentation recommends adding it), I'm adding it back with an updated value.

I added the problem without this change and the way I built it in the description https://github.com/cloudendpoints/esp/pull/870#issue-2341306665.